crates/ruff_workspace/src/configuration.rs

```diff
@@ -725,7 +761,9 @@ impl LintConfiguration {
                     warn_user_once!("The `NURSERY` selector has been deprecated.{suggestion}");
                 };
 
-                if preview.mode.is_disabled() {
+                // Only warn for the following selectors if used to enable rules
+                // e.g. use with `--ignore` or `--fixable` is okay
+                if preview.mode.is_disabled() && kind.is_enable() {
                     if let RuleSelector::Rule { prefix, .. } = selector {
                         if prefix.rules().any(|rule| rule.is_nursery()) {
                             deprecated_nursery_selectors.insert(selector);
```

Comment:
We could also push `kind` to these warning lists to facilitate different warnings by kind.